chunky/src/java/se/llbit/chunky/renderer/scene/Scene.java

```diff
@@ -1927,6 +1927,37 @@ public void postProcessPixel(int x, int y, double[] result) {
           b = QuickMath.max(0, b - 0.004);
           b = (b * (6.2 * b + .5)) / (b * (6.2 * b + 1.7) + 0.06);
           break;
+        case TONEMAP2:
+          // https://knarkowicz.wordpress.com/2016/01/06/aces-filmic-tone-mapping-curve/
+          float aces_a = 2.51f;
+          float aces_b = 0.03f;
+          float aces_c = 2.43f;
+          float aces_d = 0.59f;
+          float aces_e = 0.14f;
+          r = QuickMath.max(QuickMath.min((r * (aces_a * r + aces_b)) / (r * (aces_c * r + aces_d) + aces_e), 1), 0);
+          g = QuickMath.max(QuickMath.min((g * (aces_a * g + aces_b)) / (r * (aces_c * g + aces_d) + aces_e), 1), 0);
+          b = QuickMath.max(QuickMath.min((b * (aces_a * b + aces_b)) / (r * (aces_c * b + aces_d) + aces_e), 1), 0);
+          break;
+        case TONEMAP3:
+          // http://filmicgames.com/archives/75
+          float hA = 0.15f;
+          float hB = 0.50f;
+          float hC = 0.10f;
+          float hD = 0.20f;
+          float hE = 0.02f;
+          float hF = 0.30f;
+          r *= 16; // manually adjust exposure to approx. match other post-processing modes
```

Comment:
Indeed, they use the same exposure adjustment. I should call it "hardcoded" instead of "manual" exposure adjustment. It's a little weird because they also use it for the _tonemap1_ operator, but you don't and it still looks fine, while Hable's tonemap looks far too dark without adjusting the exposure.